### DISCUSSION TITLE
ERROR-convert_ensembl : the gene_id for ? is associated to multiple chromosomes(Pp05, Pp04). Use non ambiguous gene ids (e.g ensembl ids) please.

### DISCUSSION THREAD

**Discussion by 15132522078** [dputhier/pygtftk#181] [Q&A] [answered]
When I convert a normal GTF file to an Ensemble format GTF file, I get the above error. But I am sure that there is no such thing as gene_id duplication. Are there any other potential errors that may cause the above error?

**Comment by dputhier** [comment]
Hello,
It's hard to say without having the file. Could you provide me with the GTF file so that I can check ?
Best

**Comment by dputhier** [comment, marked as answer]
Note that it aso could be related to a format issue as the gene_id is "?". This would mean that some elements have no gene_ids.

**Reply by 15132522078** [reply]
Thank you very much for your reply. I think I've solved this problem. There are some exon comment lines(Shown below) in my gtf file that do not contain gene_id. After removing these exon comment lines, i can convert it to a GTF file in Ensemble format.
`
Pp04    maker   exon    297331  297502  .       -       .       transcript_id "PpG0000003482.01.T01";
Pp04    maker   exon    1178790 1179481 .       -       .       transcript_id "PpG0000003305.01.T01";
Pp04    maker   exon    2438500 2438536 .       -       .       transcript_id "PpG0000002963.01.T01";
Pp04    maker   exon    2438667 2438883 .       -       .       transcript_id "PpG0000002963.01.T01";
Pp04    maker   exon    2439007 2439112 .       -       .       transcript_id "PpG0000002963.01.T01";
Pp04    maker   exon    3006626 3007046 .       -       .       transcript_id "PpG0000002928.01.T03";
Pp04    maker   exon    3123882 3125305 .       +       .       transcript_id "PpG0000002826.01.T04";
Pp04    maker   exon    3124379 3125305 .       +       .       transcript_id "PpG0000002826.01.T01";
Pp04    maker   exon    3292873 3293880 .       -       .       transcript_id "PpG0000002802.01.T02";
Pp04    maker   exon    3293992 3294147 .       -       .       transcript_id "PpG0000002802.01.T01";
`
